.travis.yml

```diff
@@ -23,11 +23,20 @@ jobs:
         - scripts/kconfig/merge_config.sh .config kconfig/base-defconfig kconfig/sof-defconfig kconfig/hdaudio-codecs-defconfig
         - make modules_prepare
         - make SUBDIRS=sound/soc/sof C=2
+    - name: "coccicheck check"
+      script:
+        - export ARCH=x86_64
+        - make defconfig
+        - scripts/kconfig/merge_config.sh .config kconfig/base-defconfig kconfig/sof-defconfig kconfig/hdaudio-codecs-defconfig
+        - make modules_prepare
+        - make coccicheck MODE=report M=sound/soc/sof
```

Comment:
let's delay this one until you have a means to use Coccinelle 1.0.7, otherwise we'll have all sorts of errors